packages/backend/src/clients/EmailClient/EmailClient.test.ts

```diff
@@ -41,6 +43,8 @@ describe('EmailClient', () => {
             );
         });
         test('should create transported with secure connection when using port 465', async () => {
+            // Relies on side-effects from instancing EmailClient
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
             const client = new EmailClient({
```

Comment:
maybe we can use `client_` or `_client` don't remember which one

Reply:
Good point, will fix!